clio-util/src/main/scala/org/broadinstitute/clio/util/model/Location.scala

```diff
@@ -10,11 +10,9 @@ sealed trait Location extends EnumEntry
 object Location extends Enum[Location] {
   override val values: IndexedSeq[Location] = findValues
 
-  case object Unknown extends Location
   case object GCP extends Location
   case object OnPrem extends Location
 
   val pathMatcher = Map(GCP.toString -> GCP, OnPrem.toString -> OnPrem)
```

Comment:
I think we can get rid of `pathMatcher` too and just use `Location.namesToValuesMap` everywhere.